docs/about/policies.md

```diff
@@ -0,0 +1,101 @@
+- **Metadata access and reuse:** Metadata is licensed under CC0, except for email
+  addresses. All metadata is exported via HTTP and can be harvested.
+
+<!--
+- **Embargo status:** Users may deposit content under an embargo status and
+  provide an anticipated end date for the embargo. The repository will restrict
+  access to the data until the end of the embargo period; at which time, the
+  content will become publically available automatically. The end of the embargo
+  period is the earliest of the date provided by submitter, the first publication
+  using the data, or the end of funding support for the collection and/or disemmination
+  of the dataset.
+- **Restricted Access:** Depositors of embargoed datasets have the ability to
+  share access with other collaborators. These files will not be made publicly
+  available till the end of the embargo period.
+-->
+
+## Removal
+
+- **Revocation:** Content not considered to fall under the scope of the repository
+  will be removed and associated DOIs issued by DANDI revoked. Please signal
+  promptly, ideally no later than 24 hours from upload, any suspected policy
+  violation. Alternatively, content found to already have an external DOI will
+  have the DANDI DOI invalidated and the record updated to indicate the original
+  external DOI. User access may be revoked on violation of Terms of Use.
+<!--
+Enable after publish is in place
+- **Withdrawal:** If the uploaded research object must later be withdrawn, the
+  reason for the withdrawal will be indicated on a tombstone page, which will
+  henceforth be served in its place. Withdrawal is considered an exceptional
+  action, which normally should be requested and fully justified by the original
+  uploader. In any other circumstance reasonable attempts will be made to contact
+  the original uploader to obtain consent. The DOI and the URL of the original
+  object are retained.
+-->
+- **User data on Dandihub:** At present user data on Dandihub will be removed at
+  periodic intervals and should not be seen as persistent.
+
+## Longevity
+
+- **Versions:** Datasets are versioned when published. Prior to publishing the
+  state of a dataset may continue to evolve and the data or metadata are not
+  versioned. Derivatives of data files may be generated, but original content is
+  never modified.
+- **Replicas:** All data files are stored on an AWS public bucket, with replicas
+  housed at Dartmouth College.  Data files are kept in multiple replicas at the
+  moment, but this may change over time.
+- **Retention period:** Items will be retained for the lifetime of the repository.
+  This is currently the lifetime of the NIH award, which currently expires in
+  April 2024.
+- **Functional preservation:** DANDI makes no promises of usability and
+  understandability of deposited objects over time.
+- **File preservation:** Data files and metadata are backed up nightly and
+  replicated into multiple copies in the online system.
```

Comment:
should be somehow merged with Replicas I think. Otherwise it is just not clear what is "online system" and then contingent on "Replicas" which are disclosed to possibly change in the future